CHANGES.md

```diff
@@ -2,6 +2,8 @@
 
 #### Added
 
+- Document OCaml comment delimiters (#338, #348, @Leonidas-from-XIV)
```

Comment:
I think this can be considered a no changelog change!

Reply:
Good point. I was on the fence myself and saw that there was a change to the documentation with a changelog so I followed suit. Removed now.